---
layout: page
title: Epigenetics and Inequality
description: How do early life exposures 'get under the skin'?
img: /assets/img/inequality.jpeg
importance: 2
---

<div class="row">
    <div class="mx-auto" style="width: 250px">
        <img class="img-fluid rounded z-depth-1" src="{{ '/assets/img/inequality.jpeg' | relative_url }}" alt="" title="example image"/>
    </div>
</div>

#### Background 

Socioeconomic status (SES) is linked to health and well-being of individuals and communities. Poorer measures of SES such as social status, income, and education increase risks for depression, heart disease, inflammation, and certain cancers. Early life exposures appear to be particularly important to adult health. Notably, these effects can persist even in the context of social mobility, when individuals manage to improve in socioeconomic position over their lifetime. If, when and how individuals recover from the early life adversity associated with low SES has both theoretical and clinical health implications, but are only beginning to receive attention. 

Despite the widely documented connection between social experiences and health outcomes, the processes through which the ‘social gradient in health’ becomes biologically-embedded are still not well understood. While the range of exposures that relate to SES are myriad, a promising avenue for studying the biological ‘memory’ of SES more broadly may lie in epigenetic processes. Epigenetic processes involve mitotically- or meiotically-heritable changes in gene function that are not associated with the underlying DNA sequence itself. Epigenetic dysregulation can lead to or arise from disease through numerous pathways, and have been causally-associated with depression, heart disease, inflammatory profiles, and cancer.


I have led the bioinformatic analysis and co-written multiple papers examining the role of DNA methylation (a kind of epigenetic process) in the developmental origins of health and disease (DOHaD). In [McDade, Ryan et al. 2017](../../assets/pdf/McDade et al. - 2017 - Social and physical environments early in developm.pdf), we showed that that nutritional, microbial, and psychosocial exposures early in life predict DNAm in genes associated with inflammation measured in adulthood. 

<div class="row">
    <div class="mx-auto" style="width: 500px">
        <img class="img-fluid rounded z-depth-1" src="{{ '/assets/img/Inflammation_Volcano(3).jpg' | relative_url }}" alt="" title="example image"/>
    </div>
</div>
<div class="caption">
Volcano plots of the associations between DNAm in young adulthood and variables measured early in development, including (A) household assets in childhood, (B) extended parental absence in childhood, (C) exposure to animal feces in infancy, (D) birth in the dry season, and (E) duration of exclusive breastfeeding. Each point represents a CpG site (n = 222), with the magnitude of association between each variable and DNAm on the x axis (delta beta) and –log10 of the uncorrected P value from the linear regression models on the y axis. Blue dots represent sites with adjusted P values < 0.15.
</div>


<div class="row">
    <div class="mx-auto" style="width: 500px">
        <img class="img-fluid rounded z-depth-1" src="{{ '/assets/img/CD8A-inkscaped.jpg' | relative_url }}" alt="" title="example image"/>
    </div>
</div>
<div class="caption">
The effect of early life environment was particularly evident in some genes associated with inflammation for example CD8A. CD8A is a cell surface glycoprotein found on most cytotoxic T lymphocytes that mediates efficient cell-cell interactions within the immune system. This allows cytotoxic T cells to recognize and eliminate infected cells and tumor cells. The link between early life exposure to animal feces and this important immune associated gene may help us understand the commonly observed correlation between exposure to microbial diversity in childhood and adult chronic disease, often theorized by the hygiene hypothesis. 
</div>



In [McDade, Ryan et al. 2019](../../assets/pdf/McDade et al. - 2019 - Genome-wide analysis of DNA methylation in relatio.pdf), we expanded on our original questions about early life environments and adult health by examining the effects of socioeconomic status on DNA methylation. 




<div class="row">
    <div class="mx-auto" style="width: 500px">
        <img class="img-fluid rounded z-depth-1" src="{{ '/assets/img/hh_ll_volcano.jpg' | relative_url }}" alt="" title="example image"/>
    </div>
</div>
<div class="caption">
Volcano plot comparing low/low to high/high SES score (high/high as reference). Each point represents the difference in methylation between groups, with colored points representing significant down-methylation (red) and up-methylation (green) after accounting for false discovery (FDR q < 0.05). Taken from McDade, Ryan et al. 2019. 
</div>



We found large numbers of differentially methylated positions associated with SES, totalling 2,546 CpG sites across 1,537 annotated genes. Many of these differences were found in regions otherwise tied to cardiovascular risk and other diseases, providing a link between inequality and health.


<div class="row">
    <div class="mx-auto" style="width: 500px">
        <img class="img-fluid rounded z-depth-1" src="{{ '/assets/img/four_way_SES_venn.png' | relative_url }}" alt="" title="example image"/>
    </div>
</div>
<div class="caption">
To better understand the relative contribution of different exposures to DNAm, we broke our analyses down by commonly measured aspects of SES. Education was the single most important factor determining differences in DNAm, highlighting the importance of future research into the ways that environments and inequality 'get under the skin' and ultimately affect the health of marginalized people. 
</div>

#### Future research

We are now expanding our original analysis by incorporating participants’ whole genome (~640,000 single nucleotide polymorphisms, SNPs per individual). This will help us understand how an individual’s genetic variation (DNA) alters the way socioeconomic status affects their epigenome. Our preliminary evidence suggests that the effect of the environment is mediated in part by genetic factors, a finding that we hope will tell us more about how individuals differ in their susceptibility and resilience to the exposures of inequality.

To address population stratification and genetic variation, we used the top 2 principle components of the high-dimensional genomic dataset (Infinium Global Screening Array-24 Kit). We are now expanding our original analysis by incorporating participants’ whole genome (~640,000 single nucleotide polymorphisms, SNPs per individual). This will help us understand how an individual’s genetic variation (DNA) alters the way socioeconomic status affects their epigenome. Our preliminary evidence suggests that the effect of the environment is mediated in part by individual genetic factors, a finding that we hope will tell us more about how individuals differ in their susceptibility and resilience to the exposures of socioeconomic inequality. 
